path_x : [2.0,2.0,0.0,0.0]
path_y : [0.0,2.0,2.0,0.0]